README.md

```diff
@@ -3,9 +3,8 @@
 *HypothesisTests.jl* is a Julia package that implements a wide range of hypothesis tests.
 
 - **Build & Testing Status:**
-  [![Build status](https://github.com/JuliaStats/HypothesisTests.jl/workflows/CI/badge.svg)](https://github.com/JuliaStats/HypothesisTests.jl/actions?query=workflow%3ACI+branch%3Amaster)
-  [![Coverage Status](https://coveralls.io/repos/JuliaStats/HypothesisTests.jl/badge.svg?branch=master)
-  ](https://coveralls.io/r/JuliaStats/HypothesisTests.jl?branch=master)
+[![Build Status](https://github.com/JuliaStats/HypothesisTests.jl/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/JuliaStats/HypothesisTests.jl/actions/workflows/ci.yml)
```

Comment:
This will link to the runs on the master branch (the badge displays already only the status of the latest run on the master branch):
```suggestion
[![Build Status](https://github.com/JuliaStats/HypothesisTests.jl/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/JuliaStats/HypothesisTests.jl/actions/workflows/ci.yml?query=branch%3Amaster)
```

Reply:
Ah thanks I wanted this but GitHub didn't propose it.